packages/blocks/src/multiple-choice-question/index.js

```diff
@@ -4,21 +4,35 @@
 import { RichText } from '@wordpress/block-editor';
 import { createContext } from '@wordpress/element';
 import classnames from 'classnames';
+import { isNil } from 'lodash';
 
 /**
  * Internal dependencies
  */
-import { QuestionHeader, QuestionWrapper } from '../components';
+import { ErrorMessage, QuestionHeader, QuestionWrapper } from '../components';
 import { Style } from './constants';
+import { useField } from '@crowdsignal/form';
 
 const Context = createContext();
 
 const MultipleChoiceQuestion = ( { attributes, children, className } ) => {
+	const { error } = useField( {
+		name: `q_${ attributes.clientId }[choice]${
+			attributes.maximumChoices !== 1 ? '[]' : ''
+		}`,
+		validation: ( value ) => {
+			if ( attributes.mandatory && ( value === '' || isNil( value ) ) ) {
+				return 'This question is required';
+			}
+		},
+	} );
+
```

Comment:
We may want to refactor this a bit to avoid code repetition on the `MultipleChoiceAnswer`.
Currently, the `useField` hook is used on each answer but, in order to validate it properly, we need to `useField` on the question or pass the `error` status from the answers to the question.

Any thoughts about this one?

Reply:
I've created an `useValidation` hook to be used on the MC Question and also updated the `useField` hook to use it internally.
Please take a look and let me know if the implementation matches what you're suggesting.